docs/notebooks/how_bambi_works.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>It used to be the case that <code>az.style.use("arviz-darkgrid")</code> (the same for any matplotlib style) has to be defined in a different cell than the one used to import matplotlib. Otherwise only part of the style was actually applied. Not sure if this is still the case </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>I wasn't aware of this. Maybe related to this is that sometimes I had to use <code>%matplotlib inline</code> if I wanted styles to apply. Weird Jupyter magic.  </p>